controllers/ibu_controller.go

```diff
@@ -120,26 +120,16 @@ func (r *ImageBasedUpgradeReconciler) Reconcile(ctx context.Context, req ctrl.Re
 
 	currentInProgressStage := utils.GetCurrentInProgressStage(ibu)
 	if currentInProgressStage != "" {
-		nextReconcile, err = r.handleStage(ctx, ibu, currentInProgressStage)
```

Comment:
Why dropping the last arg? It can be different from what's in spec.

Reply:
fixed